lib/shared/src/models/index.test.ts

```diff
@@ -1,35 +1,35 @@
 import { beforeAll, describe, expect, it } from 'vitest'
 import { ModelProvider } from '../models/index'
-import { DEFAULT_CHAT_MODEL_TOKEN_LIMIT, tokensToChars } from '../prompt/constants'
 import { DOTCOM_URL } from '../sourcegraph-api/environments'
+import { CHAT_TOKEN_BUDGET } from '../token/constants'
 import { DEFAULT_DOT_COM_MODELS } from './dotcom'
 import { ModelUsage } from './types'
 
-describe('getMaxCharsByModel', () => {
+describe('getMaxTokenByID', () => {
     beforeAll(() => {
         ModelProvider.getProviders(ModelUsage.Chat, false, DOTCOM_URL.toString())
     })
 
     it('returns default token limit for unknown model', () => {
-        const maxChars = ModelProvider.getMaxCharsByModel('unknown-model')
-        expect(maxChars).toEqual(tokensToChars(DEFAULT_CHAT_MODEL_TOKEN_LIMIT))
+        const maxToken = ModelProvider.getMaxTokenByID('unknown-model')
```

Comment:
Let's rename `getMaxTokenByID` to be `maxRequestTokensForModel` or something like that?

Reason is the tokens also have numbers, and "max token" sounds like the largest token value.
```suggestion
        const maxTokens = ModelProvider.maxRequestTokensForModel('unknown-model')
```